components/extractors/database/index.md

```diff
@@ -25,12 +25,13 @@ There are a number of extractors for [**SQL databases**](/components/extractors/
 - [Teradata](https://www.teradata.com/)
 
 They are all [configured](/components/extractors/database/sqldb/#create-new-configuration) in the same way and 
-have an [advanced mode](/components/extractors/database/sqldb/). *(Their basic configuration is also part 
-of our [tutorial](/tutorial/load/database/).)* 
+have an [advanced mode](/components/extractors/database/sqldb/). 
```

Comment:
jeste to prihod prosim do toho seznamu nahore (to je kvuli seo :) )

Reply:
ok. vdaka.

Reply:
do zoznamu hore to uplne dat nemoze, lebo to je zoznam SQL databaz, ktore maju spolocne rozhranie .... Cosmos je JSON NoSQL db, ktora ma SQL API (ale to nie je plnohodnotne SQL, nedaju sa robit JOINs) ... 

Preformuloval som to aby tam boli aj linky na samotne databazy, pozri prosim, ci moze byt ... nic lepsie mi nenapada.

Reply:
jo, dobry!